docs/en/stack/ml/df-analytics/dataframeanalytics.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[ml-dfanalytics-overview]]
+== {dfanalytics-cap} overview
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+{dfanalytics-cap} allows you to perform different analyses of your data and 
```

Comment:
I stole this introductory sentence for the following page: https://www.elastic.co/guide/en/elastic-stack-overview/master/ml-dfanalytics.html
Should all of the content in this introductory section be moved to that page (akin to what was done in the Data frame transforms page)? If so, "Outlier detection" would be the first sub-page, which seems okay to me.